srcpkgs/vdirsyncer/template

```diff
@@ -1,23 +1,24 @@
 # Template file for 'vdirsyncer'
 pkgname=vdirsyncer
-version=0.18.0
-revision=3
+version=0.19.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools_scm"
 depends="python3-atomicwrites python3-click python3-click-log
- python3-click-threading python3-requests-toolbelt"
-checkdepends="python3-pytest python3-hypothesis $depends"
+python3-click-threading python3-requests-toolbelt python3-aiohttp
+python3-aiostream"
+checkdepends="python3-pytest python3-hypothesis python3-aioresponses python3-trustme python3-pytest-asyncio python3-pytest-localserver python3-pytest-httpserver $depends"
 short_desc="Synchronize calendars and addressbooks"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://vdirsyncer.pimutils.org/"
 distfiles="${PYPI_SITE}/v/vdirsyncer/vdirsyncer-${version}.tar.gz"
-checksum=27bc3ed51f774935fbba392915c8c8d4cf639ae51a44b674686b49a1025fc201
+checksum=8e1e8403a08659e5a4e7fa3e9caaa2e2dce2bf1f98d923029049a34db75a2525
 
 do_check() {
-	# Disable coverage checks and those requiring unpackaged pytest-localserver
+# Disable coverage checks and those requiring unpackaged pytest-localserver
 	vsed -e 's/^addopts/noaddopts/' -i setup.cfg
-	PYTHONPATH="$(cd build/lib* && pwd)" \
+		PYTHONPATH="$(cd build/lib* && pwd)" \
 		python3 -m pytest -k 'not test_request_ssl_fingerprints'
 }
```

Comment:
This whole function can be removed and the package will just use the default one from the build style.